Changelog.md

```diff
@@ -22,11 +22,13 @@ Breaking changes:
  * Source mappings: Add "modifier depth" as a fifth field in the source mappings.
  * AST: Inline assembly is exported as structured JSON instead of plain string.
  * General: ``private`` cannot be used together with ``virtual``.
+ * Inheritance: State variable shadowing is now disallowed.
 
 Language Features:
  * Allow global enums and structs.
  * Allow underscores as delimiters in hex strings.
  * Allow explicit conversions from ``address`` to ``address payable`` via ``payable(...)``.
+ * Inheritance: Disallow state variables to shadow state variables from base contracts.
```

Comment:
Isn't one entry enough?

Reply:
Should this one be removed?

Reply:
Yep - it's a breaking change, I'd say that's enough and that's what we did for all others, right (haven't checked)?